docs/config-api.md

```diff
@@ -175,6 +175,13 @@ System.config({
 });
 ```
 
+* `format`: Sets in what format the module is loaded. See [Module Formats](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md).
+* `exports`: Used to create a global variable, pointing to this meta path. Only available at the time of executing the module referenced by meta path. When meta is used inside `packages` configuration this variable is only available within that package. More info: [Exports](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#exports)
```

Comment:
"For `global` modules, tells the loader what global export name to use as the module value". Also, does not have any exceptions for packages configuration at all.